src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.test.tsx

```diff
@@ -247,4 +249,18 @@ describe("ResultsBlock", () => {
     expect(dateComponent).toBeVisible();
     expect(dateComponent).toHaveTextContent(relativeTimeFromPastDate);
   });
+
+  it("should show message when there is no result", () => {
+    state.juju.crossModelQuery.loaded = true;
+    state.juju.crossModelQuery.results = null;
+    renderComponent(<ResultsBlock />, { state });
+    expect(screen.getByText("No results returned!")).toBeVisible();
+  });
+
+  it("should show message when all models have no result", () => {
+    state.juju.crossModelQuery.loaded = true;
+    state.juju.crossModelQuery.results = { mockModel0: [], mockModel1: [null] };
+    renderComponent(<ResultsBlock />, { state });
+    expect(screen.getByText("No results returned!")).toBeVisible();
```

Comment:
Would you mind converting the `"No results returned!"` text to a Label enum so that if we change the message these tests continue to work?

Reply:
Done!